src/plugins/process/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+include (LibAddMacros)
+
+find_package (Pluginprocess)
+
+if (PLUGINPROCESS_FOUND)
```

Comment:
Isn't dump needed, too?

Reply:
yes thats what i thought infos/needs takes care about, will correct it